hashicorp/dev-portal
# Remove Waypoint API docs

## 🔗 Relevant links
- [Preview link](https://dev-portal-git-heat-chorewaypoint-ia-remove-api-docs-hashicorp.vercel.app/waypoint/docs) 🔎
- [Asana task](https://app.asana.com/0/1206225787851796/1206707955223698/f) 🎟️

## 🗒️ What

- Remove Waypoint API docs
- Add redirect to swagger doc
- Remove API from Waypoint docs landing page and top level nav

## 🤷 Why

- Part of the Waypoint IA restructure

## 🧪 Testing

- visit the [Waypoint docs landing page in prod](https://developer.hashicorp.com/waypoint/docs)
- visit the [Waypoint docs landing preview page](https://dev-portal-git-heat-chorewaypoint-ia-remove-api-docs-hashicorp.vercel.app/waypoint/docs)
  - See that the API  card is gone and only the CLI card is visible
  - see that API is gone from the top level nav

<img width="1081" alt="Screenshot 2024-02-27 at 15 53 43" src="https://github.com/hashicorp/dev-portal/assets/55773810/cbceaa9e-bd61-45d6-8e44-ff019ceece48">


## 💭 Anything else?

nay
